README.md

```diff
@@ -1,3 +1,96 @@
+# Cadence CLI
+
+## Install
+```shell
+pip install jetbrains-cadence
+```
+
+This will create a `cadence` script in your current environment.
+
+## Getting Started
+
+```shell
+cadence login
+```
+If you want to use Cadence CLI in the non-interactive environment, you can [create the token manually](https://api.cadence.jetbrains.com/app/jettrain/token.html) and pass it via `CADENCE_TOKEN` environment variable
+
+
+#### Start the execution from YAML config
+```shell
+cadence execution start --preset path/to/config.yaml
+```
+This will print the ID of the started execution.
+
+
+#### See the execution status
+```shell
+cadence execution status <YOUR-EXECUTION-ID>
+```
+
+#### Stop the execution
+```shell
+cadence execution stop <YOUR-EXECUTION-ID>
+```
+
+## More commands
+
+#### Get information about the execution as a JSON
+```shell
+cadence execution info <YOUR-EXECUTION-ID>
+```
+
+#### List executions
+```shell
+cadence execution list
+```
+Options:
+```
+  --offset INTEGER  [default: 0]
+  --count INTEGER   [default: 50]
+  --all             List all executions. Count and offset are ignored
+  --json / --table  Output format  [default: table]
+```
+
+#### View execution logs
+```shell
+cadence execution logs <YOUR-EXECUTION-ID>
+```
+
+
+#### Open terminal
+```shell
+cadence execution terminal <YOUR-EXECUTION-ID>
+```
+
+#### Download data
+```shell
+cadence execution download <YOUR-EXECUTION-ID>
+```
+Options:
+```
+  --to DIRECTORY  [required]
+  --inputs        Include inputs
+  --no-outputs    Exclude outputs
+```
+
+## Workspace management
+#### Display information about the current workspace:
+```shell
+cadence workspace
+```
+
+#### See available workspaces:
+```shell
+cadence workspace list
+```
+
+#### Set workspace:
+```shell
+cadence workspace set <YOUR-WORKSPACE-ID>
+```
+
+
+
 ## Config file format:
```

Comment:
Let's add a section Default storage before "Custom storage" and explain that it's an s3 storage provided by the service and so on

inspiration: https://plugins.jetbrains.com/plugin/23731-jetbrains-cadence/edit/custom-pages/docs/storages.html#built_in_storage